classes/core/PKPContainer.php

```diff
@@ -286,6 +322,7 @@ public function registerCoreContainerAliases()
     protected function loadConfiguration()
     {
         $items = [];
+        $_request = new \APP\core\Request;
```

Comment:
I'm not sure instantiating a new `Request` is going to work well; I would recommend using `PKPApplication::get()->getRequest()` instead.

Reply:
Updated but I don't think this has any different effect as this was done at initialization of the app (in the constructor) so both way we get same instance return back .